### ISSUE TITLE
nodeColor paints duped color

### ISSUE THREAD

**Issue by sinramyeon** [amcharts/amcharts4#2381]
in the colouring example in https://www.amcharts.com/docs/v4/chart-types/chord-diagram/#Node_color_via_color_data_field

```
chart.data = [
  { "from": "A", "to": "D", "value": 10, "nodeColor": "#93B5C6" },
  { "from": "B", "to": "D", "value": 8, "nodeColor": "#DDEDAA" },
  { "from": "B", "to": "E", "value": 4, "nodeColor": "#DDEDAA" },
  { "from": "B", "to": "C", "value": 2, "nodeColor": "#DDEDAA" },
  { "from": "C", "to": "E", "value": 14, "nodeColor": "#F0CF65" },
  { "from": "E", "to": "D", "value": 8, "nodeColor": "#D7816A" },
  { "from": "C", "to": "A", "value": 4, "nodeColor": "#F0CF65" },
  { "from": "G", "to": "A", "value": 7, "nodeColor": "#BEC5AD" },
  { "from": "D", "to": "B", "value": 1, "nodeColor": "#BD4F6C" }
];

chart.dataFields.fromName = "from";
chart.dataFields.toName = "to";
chart.dataFields.value = "value";
chart.dataFields.color = "nodeColor";
```

nodecolor A and D is different but from A to D causes D to paint same colour as A.

**Comment by martynasma** [comment]
Seems like that doc article was a bit outdated. Thanks for bringing it to our attention!

I went ahead and updated the tutorial with how to apply the colors properly:

https://www.amcharts.com/docs/v4/chart-types/chord-diagram/#Node_color_via_color_data_field

TL;DR version:

```JavaScript
chart.data = [
  // Define colors
  { "from": "A", "nodeColor": "#93B5C6" },
  { "from": "B", "nodeColor": "#DDEDAA" },
  { "from": "C", "nodeColor": "#F0CF65" },
  { "from": "D", "nodeColor": "#BD4F6C" },
  { "from": "E", "nodeColor": "#D7816A" },
  { "from": "G", "nodeColor": "#BEC5AD" },
  
  // Actual data
  { "from": "A", "to": "D", "value": 10 },
  { "from": "B", "to": "D", "value": 8 },
  { "from": "B", "to": "E", "value": 4 },
  { "from": "B", "to": "C", "value": 2 },
  { "from": "C", "to": "E", "value": 14 },
  { "from": "E", "to": "D", "value": 8 },
  { "from": "C", "to": "A", "value": 4 },
  { "from": "G", "to": "A", "value": 7 },
  { "from": "D", "to": "B", "value": 1 }
];
```
